server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/IMAPServerFactory.java

```diff
@@ -66,7 +68,7 @@ public IMAPServerFactory(FileSystem fileSystem, ThrowingFunction<HierarchicalCon
         this.connectionCheckFactory = connectionCheckFactory;
     }
 
-    protected IMAPServer createServer(HierarchicalConfiguration<ImmutableNode> config) {
+    private IMAPServer createServer(HierarchicalConfiguration<ImmutableNode> config) {

```

Comment:
Changing `protected` to `private` risks breaking forked projects that are attempting to reuse this function.